funnel/views/schedule.py

```diff
@@ -143,10 +143,11 @@ class ProjectScheduleView(ProjectViewMixin, UrlForView, ModelView):
     @render_with('schedule.html.jinja2')
     @requires_permission('view')
     def schedule(self):
-        return dict(project=self.obj, venues=self.obj.venues,
+        return dict(project=self.obj,
             from_date=date_js(self.obj.date), to_date=date_js(self.obj.date_upto),
             sessions=session_list_data(self.obj.scheduled_sessions, with_modal_url='view_popup'),
             timezone=timezone(self.obj.timezone).utcoffset(datetime.now()).total_seconds(),
+            venues=[dict(venue.current_access()) for venue in self.obj.venues],
```

Comment:
Is the `dict(*.current_access())` necessary? It's unnecessary verbosity, so if something is breaking in the absence of the `dict` wrapper, we should fix it.

Reply:
@jace the template runs `{{ venues|tojson }}` on it. `current_access()` isn't JSON serializable without `dict()`.

Reply:
We can fix this in Baseframe by replacing the function Jinja2 uses: http://jinja.pocoo.org/docs/2.10/api/#policies

I will file a separate ticket for this.

Reply:
Flask does its own thing. It's easier to override. http://flask.pocoo.org/docs/1.0/api/#flask.Flask.json_encoder